scripts/generate-react-components.js

```diff
@@ -8,6 +8,19 @@ const svgr = require('@svgr/core').default;
 const Case = require('case');
 const buble = require('buble');
 
+const reactUtils = `import { createContext, useContext, useState } from 'react';
+
+const uniqueIdPrefixContext = createContext('__GeneratedJBProductLogos__');
+
+export const UniqueLogosIdPrefixProvider = uniqueIdPrefixContext.Provider;
+
+let i = 0;
+export const useUniqueId = () => {
+  const prefix = useContext(uniqueIdPrefixContext);
+  const [ id ] = useState(() => prefix + i++);
```

Comment:
there's probably no point in using state here, as it never changes

Reply:
It's for decrease counter `i` iteration speed. In this way it iterates only when component which uses a hook was mounted.